assets/src/containers/Course.js

```diff
@@ -47,54 +49,68 @@ function Course (props) {
               courseId={courseId}
               courseInfo={courseInfo}
             />
-            <Route
-              path='/courses/:courseId/'
-              exact
-              render={props =>
-                <IndexPage
-                  {...props}
-                  courseInfo={courseInfo}
-                  courseId={courseId}
-                />}
-            />
-            <Route
-              path='/courses/:courseId/grades'
-              render={props =>
-                <GradeDistribution
-                  {...props}
-                  disabled={!courseInfo.course_view_options.gd}
-                  courseId={courseId}
-                  user={user}
-                />}
-            />
-            <Route
-              path='/courses/:courseId/assignmentsv1'
-              render={props =>
-                <AssignmentPlanning
-                  {...props}
-                  disabled={!courseInfo.course_view_options.apv1}
-                  courseId={courseId}
-                />}
-            />
-            <Route
-              path='/courses/:courseId/assignments'
-              render={props =>
-                <AssignmentPlanningV2
-                  {...props}
-                  disabled={!courseInfo.course_view_options.ap}
-                  courseId={courseId}
-                />}
-            />
-            <Route
-              path='/courses/:courseId/resources'
-              render={props =>
-                <ResourcesAccessed
-                  {...props}
-                  disabled={!courseInfo.course_view_options.ra}
-                  courseInfo={courseInfo}
-                  courseId={courseId}
-                />}
-            />
+            {courseInfo.course_user_exist === 0
+             ? (
+                <>
+                  <WarningBanner>
+                    No data is available for {courseInfo.name} yet. Please wait for next system data load.
+                  </WarningBanner>
```

Comment:
Were you going to change this to `AlertBanner`?

Reply:
Changed it to AlertBanner.

Reply:
Updated to AlertBanner.